README.md

```diff
@@ -1,2 +1,223 @@
-# Microverse-Capstone-M1
-It is A Capstone Project in HTML, CSS, and JavaScript Assigned by Microverse in Outcome Based Assessment.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
[OPTIONAL]
- [ ] Kindly remove all comments from the readme file. They are just instructions on how you should write a professional README file.

Reply:
done.